src/app/components/task-list/task-list.component.ts

```diff
@@ -16,20 +16,15 @@ import { RemoveButtonComponent } from 'src/app/remove-button/remove-button.compo
 })
 export class TaskListComponent implements OnInit, OnDestroy {
   completedTasksSubscription = Subscription.EMPTY;
-  exampleTasksSubscription = Subscription.EMPTY;
   tasksSubscription = Subscription.EMPTY;
   tasksCompleted: number = 0;
   tasks: Task[] = [];
 
   constructor(private http: HttpClient, private taskService: TaskService) {}
 
   ngOnInit(): void {
-    this.exampleTasksSubscription = this.taskService.getExampleTasks().subscribe((tasks) => {
-      if(!this.taskService.initialized){
-        this.taskService.setTasks(tasks);
-      } 
-    })
-    this.tasksSubscription = this.taskService.tasksSubject.subscribe((tasks) => {
+    this.tasksSubscription = this.taskService.getTasks().subscribe((tasks) => {
+      this.taskService.setTasks(tasks);
```

Comment:
if you set tasks in the service then it would make sense to use that tasks state everywhere you need, this component included you can remove the line below this comment as it won't be necessary after these changes